internal/pkg/aws/ec2/ec2.go

```diff
@@ -454,8 +454,16 @@ func (c *EC2) GetManagedPrefixListId(prefixListName string) (*string, error) {
 		return nil, fmt.Errorf("query returned error: %w", err)
 	}
 
-	if len(prefixListOutput.PrefixLists) != 1 {
-		return nil, fmt.Errorf("query `%s` did not return one prefix list", prefixListName)
+	if len(prefixListOutput.PrefixLists) == 0 {
+		return nil, fmt.Errorf("cannot find any prefix list with name: %s", prefixListName)
+	}
+
+	if len(prefixListOutput.PrefixLists) > 1 {
+		var ids []*string
+		for _, v := range prefixListOutput.PrefixLists {
+			ids = append(ids, v.PrefixListId)
+		}
```

Comment:
nit: we could do this out side of the `if` and check `ids`. If the length is 1 then we return `ids[0]`